### PR TITLE
fix(lyrics-plus): absolute positioning in container

### PR DESCRIPTION
**Pull request by ohitstom**
<details open>
<summary>Visual</summary>

<table>
  <tr>
    <td>
      Image 1
    </td>
    <td>
      Image 2
    </td>
  </tr>
</table>

</details>

<details open>
<summary>Technical</summary>

<table>
  <tr>
    <td>
      Image 3
    </td>
    <td>
      Image 4
    </td>
  </tr>
</table>

</details>

removes scrollbar etc, please test before accepting however i did go over everything i could think of (fullscreen, no blur/overlay, no lyrics, different text sizes, compact vs non compact etc.)

### CODE REVIEW

**Comment by ohitstom** [comment]
@kyrie25 @rxri

File: CustomApps/lyrics-plus/style.css [hunk 1]
```diff
@@ -89,7 +90,7 @@
 
 .lyrics-lyricsContainer-LyricsUnsyncedPadding {
 	display: flex;
-	height: 25vh;
+	height: 10vh;
```
Comment (kyrie25) [inline CustomApps/lyrics-plus/style.css:93]: Doesn't look quite right when the track is instrumental, should leave it as-is or find another solution.
image
image

**Review by kyrie25 — CHANGES_REQUESTED** [review]

File: CustomApps/lyrics-plus/style.css (hunk 1 above)
Comment (kyrie25) [reply CustomApps/lyrics-plus/style.css:93]: Doesn't look quite right when the track is instrumenta.
image

```suggestion
	height: calc(50vh - 12px - 90px - var(--lyrics-font-size));
}

.lyrics-lyricsContainer-UnsyncedLyricsPage:has(.lyrics-lyricsContainer-LyricsLine:nth-child(3)) .lyrics-lyricsContainer-LyricsUnsyncedPadding {
	height: 10vh;
}
```

**Review by kyrie25 — CHANGES_REQUESTED** [review]
Address comment.

**Review by kyrie25 — APPROVED** [review]

**Review by theRealPadster — APPROVED** [review]

**Review by rxri — APPROVED** [review]

**Pull request merged**